src/Crowdin.Api/Branches/BranchesSortingRule.cs

```diff
@@ -0,0 +1,31 @@
+
+using System.ComponentModel;
+using JetBrains.Annotations;
+
+namespace Crowdin.Api.Branches
+{
+    [PublicAPI]
+    public enum BranchesSortingRule
```

Comment:
It looks like this does not support complex ordering like `orderBy=createdAt desc,name,priority`